Created security page (#3103)
OpenShot/openshot-qt: SECURITY.md

# Security Policy

## Supported Versions

> All versions of Openshot are security supported. This means we keep all versions of Openshot update to date with the latest security features around.

Reviewer: Just out of curiosity — devil's advocate, here — what is that claim based on? What facts provide the basis for it? As we say on Wikipedia, `[citation needed]`.

> All versions of Openshot are security supported. This means we keep all versions of Openshot update to date with the latest security features around.

Reviewer: ```suggestion
All versions of OpenShot are security supported. This means we keep all versions of OpenShot update to date with the latest security features around.
```

(Also, "all versions"? Not only is that not true, it _shouldn't_ be true. If someone finds a security vulnerability in OpenShot 2.1.0, the correct response is, "That version is no longer supported, as it's out of date. Upgrade to a newer version." There is absolutely no way anyone's going to fix an OpenShot 2.1.0 security issue. So, given that _all_ versions are **not** supported, the question becomes: Which ones _are_. Which versions, concretely, are developers willing to **promise** they'll update in response to vulnerability notifications?

## Reporting a Vulnerability

> Use this section to tell people how to report a vulnerability.

Reviewer: Pretty sure this line was supposed to be edited out.

> If you find a security vulnerability we would like to here from you asap.

Reviewer: ```suggestion
If you find a security vulnerability we would like to hear from you as soon as possible.
```

(It's a bad idea to assume that colloquialisms like "asap" are universally understood.)

> Keeping Openshot and our users safe is very important to us.

Reviewer: ```suggestion
Keeping OpenShot and our users safe is very important to us.
```

If you do find a security vulnerability you can report it to jonathan@openshot.org.
It should take us a day to the latest a week to get back to you. 

> If we do comfirm it to be a real ssecurity vulnerability we will get a patch out hopefully asap. 

Reviewer: ```suggestion
If we confirm the reported vulnerability exists in OpenShot, we will get a patch out in a reasonable timeframe.
```

(In an ideal world this would be a concrete timeframe, "within 30 days" or "in the next release, no later than 6 weeks" or whatever. But this is a largely-volunteer effort, and commitments like that are meaningless if there isn't someone who can **promise** they'll be held to. ...Failing that, non-committal commitments are understandable. But "hopefully asap" is not a response commitment.)

> Thank you for taking the time to care about Openshot and the saftey of it.

Reviewer: ```suggestion
Thank you for taking the time to care about OpenShot and its users.
```
("Safety" is a loaded word, it's impossible to define in these contexts. Say your email account is hacked. Does that threaten your _safety_, necessarily? Well, it **might**, or it might not, in a lot of different ways. Depends on who hacked it, what they saw, and what they intend to do with it. Etc...)
